user-guide/Troubleshooting/Known_issues/KI_Problem_after_OS_upgrade_to_win_2022.md

```diff
@@ -19,23 +19,7 @@ After an upgrade to Windows Server 2022, the following issues can cause problems
 
 - For the Netlogon issue, refer to the Microsoft article [Netlogon service does not keep settings](https://learn.microsoft.com/en-us/troubleshoot/windows-server/active-directory/netlogon-service-not-start-automatically)
```

Comment:
Following the steps in this link is not enough, for now we still need the "IIS 6 Management Compatibility" feature as well.

Reply:
Could you give me some more information about this so I can add this too? What exactly does the user need to do for this?

Reply:
I put this comment in the wrong place. Before running ConfigureIIS.bat, it's still needed to install IIS first. This can be done using the "Server Manager", enable IIS, and next to the default selections also enable the following features: WebSockets and "IIS 6 Management Compatibility".

Reply:
Updated!